example/src/main/java/com/stripe/example/module/StripeIntentViewModel.kt

```diff
@@ -23,16 +26,45 @@ internal class PaymentIntentViewModel(
     fun createPaymentIntent(
         country: String,
         callback: (JSONObject) -> Unit
+    ) {
+        callApi(
```

Comment:
Maybe rename this `makeBackendRequest`? It's not directly calling the API (your backend is)

Reply:
fair